### DISCUSSION TITLE
CAT1 API vs Coloured Coin api

### DISCUSSION THREAD

**Discussion by ByronAP** [Chia-Network/chia-blockchain#8689] [Q&A]
With Bram announcing that Chia "coloured coins" will be called CAT1 (Chia Asset Token) (https://www.chia.net/2021/09/23/chia-token-standard-naming.en.html) does this mean the cc api calls will be scraped?

**Comment by hoffmang9** [comment]
There will be some changes which you can review here - https://github.com/Chia-Network/chia-blockchain/compare/protocol_and_cats_rebased

**Reply by ByronAP** [reply]
It appears that the naming of the API calls will still use the CC moniker. I think that is/will be a point of confusion and lead to issues down the line if not changed.

**Comment by BrandtH22** [comment]
@ByronAP , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
